src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.cs

```diff
@@ -37,7 +37,11 @@ public void TestRecoveryServicesVaultCRUD()
             TestController.NewInstance.RunPsTest(_logger, "Test-RecoveryServicesVaultCRUD");
         }
 
+#if NETSTANDARD
+        [Fact(Skip = "Different parameter set used for NetStandard. Cannot process command because of one or more missing mandatory parameters: Certificate.")]
```

Comment:
Let's file an issue for them to fix this.

Reply:
Created: https://github.com/Azure/azure-powershell/issues/7534